docs/src/ComponentsPage.js

```diff
@@ -574,6 +574,24 @@ const ComponentsPage = React.createClass({
                   <code>getValue()</code> will not work when used this way.</p>
                   <ReactPlayground codeText={Samples.InputWrapper} />
                 </div>
+
+                {/* Menuitem */}
+                <div className='bs-docs-section'>
+                  <h1 id='menu-item' className='page-header'>MenuItem</h1>
+                  <p>This is a component used in other components (see <a href="buttons">Buttons</a>, <a href="#navbars">Navbar</a>).</p>
+                  <p>It supports the basic anchor properties <code>href</code>, <code>target</code>, <code>title</code>.</p>
+                  <p>It also supports different properties of the normal Bootstrap MenuItem.
+                    <ul>
+                      <li><code>header</code>: To add a header label to sections</li>
+                      <li><code>divider</code>: Adds an horizontal divider between sections</li>
+                      <li><code>disabled</code>: shows the item as disabled, and prevents the onclick</li>
+                      <li><code>eventKey</code>: passed to the callback</li>
+                      <li><code>onSelect</code>: a callback that is called when the user clicks the item.</li>
+                    </ul>
+                  <p>The callback is called with the following arguments: <code>eventKey</code>, <code>href</code> and <code>target</code></p>
+                  </p>
+                  <ReactPlayground codeText={Samples.MenuItem} />
+                </div>
```

Comment:
I wonder about positioning of this section. Should we put this next to either the dropdown buttons or nav sections?